js/main.js

```diff
@@ -85,6 +85,34 @@ MINUS.work = (function($) {
     }
   }
 
+  setInterval(function(){
+    $('.line-chart span').addClass('is-active');
+  }, 500);
+
+
+  var animation_elements = $('.work-wrap');
```

Comment:
🐍 _case should only be used in ruby or php (?!)... we really need to start a "styleguide" and bring some linter rules to our projects.